config/initializers/schema_format.rb

```diff
@@ -1,4 +1,6 @@
 # This is just for running test.
 # This isn't used normal use case.
 # Redmine doesn't use plugins/*/config/initializers/*.rb
-ActiveRecord::Base.schema_format = :sql
+# `ActiveRecord::Base.schema_format=` is removed from Rails v7 but maintained for backward compatibility.
+active_record = ActiveRecord.respond_to?(:schema_format=) ? ActiveRecord : ActiveRecord::Base
+active_record.schema_format = :sql
```

Comment:
Can we use `Rails.application.config.active_record.schema_format = :sql` instead?

Reply:
I tried to use it but It didn't work because the files under `config/initialziers` would seem to be loaded after that Rails application loaded every configurations in `config/application.rb` and `config/environments/*`
- ref: https://railsguides.jp/configuring.html#%E3%82%A4%E3%83%8B%E3%82%B7%E3%83%A3%E3%83%A9%E3%82%A4%E3%82%B6%E3%83%95%E3%82%A1%E3%82%A4%E3%83%AB%E3%82%92%E4%BD%BF%E3%81%86

Reply:
Oh... OK. Let's set it directly.

Could you simplify this something like the following?

```ruby
if ActiveRecord.respond_to?(:schema_format=)
  # For Rails >= 7 
  ActiveRecord.schema_format = :sql
else
  ActiveRecord::Base.schema_format = :sql
end
```

Reply:
Sure. And also I will delete the following comment for simplicity.
```ruby
# `ActiveRecord::Base.schema_format=` is removed from Rails v7 but maintained for backward compatibility.
```

Reply:
fix: [c0543e8](https://github.com/clear-code/redmine_full_text_search/pull/128/commits/c0543e8a874ca5220eec644b4ba83c9d1b911681) I understood the previous code was hard to read because users couldn't see the main class like ActiveRecord or ActiveRecord::Base.